specify/specify7
# Add geo trees to tree creation tool

Fixes #5371

### Checklist

      and self-explanatory (or properly documented)

### Testing instructions

- open tree viewer 
- click add a new tree 
- [ ] verify you can add each of the 3 new trees: Rocks - Minerals - Meteorites
